templates/qemu.conf

```diff
@@ -0,0 +1,5 @@
+<% if @qemu_conf -%>
```

Comment:
there is no need for that, since we only use this template if @qemu_conf is not {} (which is great btw.).
and if qemu_conf is {} then the result will be empty anyway.